.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.3
+current_version = 1.0.5
```

Comment:
Looks like the rebase has pulled in some other commits? Shouldn't be any bumpversion things here.